source/reference/minio-mc-admin/mc-admin-trace.rst

```diff
@@ -213,6 +213,11 @@ Syntax
 
    Returns calls matching the supplied request header.
 
+.. mc-cmd:: --request-query
+
+   Returns calls matching the supplied request query parameter.
+   This debug option should only be used at the direction of MinIO Support.
```

Comment:
Went with minimalist option: in docs, but discouraged without fanfare. It won't mean much to folks outside MinIO Engineering.

Is `request-header` the same situation? Or is that one useful for customers.

Reply:
Hard to say - we can chat w/ engineering on this to understand, and maybe ask for a larger pass on what params we want to mark as 'leave this knob alone'

Reply:
Will follow up with eng about additional concerns. Getting this update out in the meantime.